osmaxx/api_client/API_client.py

```diff
@@ -4,6 +4,8 @@
 
 logger = logging.getLogger(__name__)
 
+HTTPError = requests.HTTPError
```

Comment:
Thinking a bit more about this:
This seems for me the wrong encapsulation; we don't control what is being raised, we just assume that `raise_for_status` is actually raising a `HTTPError`. This is very implicit, so we either should re-raise it with our own Exception-Class (or wrapper thereof) or just drop the whole game: our class raises another class' exception, that is nothing uncommon. We are relying very heavily on the requests library, no need to hide that away.